package.json

```diff
@@ -1080,6 +1090,16 @@
                                 "pattern": "^(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])$",
                                 "type": "string"
                             },
+                            "portRangeStart": {
+                                "default": null,
```

Comment:
Set it to a proper default.

Reply:
I think `fixedPortRangeStart` default value of -1 is being used to see if a user wants a fixed one or a scanned one. And, it takes precedence over `portScanRangeStart`. May I recommend using type to be
`"type": ["integer", "null"]`
or using the `anyOf` and default can be null. And, I think you can still have reasonable min/max in case of an integer.

The so-called spec I found here https://cswr.github.io/JsonSchema/spec/multiple_types/

Reply:
Have changed this also